tests/TestInstall/CMakeLists.txt

```diff
@@ -40,9 +49,11 @@ endif()
 if(SURELOG_USE_HOST_UHDM)
   find_package(UHDM REQUIRED)
   find_package(CapnProto)
+  set(UHDM_LIBRARY uhdm::uhdm)
   set(CAPNPROTO_LIBRARY CapnProto::capnp)
 else()
   set(UHDM_LIBRARY uhdm)
+  set(UHDM_LIB_DIR "${INSTALL_DIR}/lib/uhdm")
```

Comment:
On my system, the library ends up in lib64/, so a local hack of
```
set(UHDM_LIB_DIR "${INSTALL_DIR}/lib/uhdm" "${INSTALL_DIR}/lib64/uhdm")
```

does work. But maybe there are better ways ?

Reply:
it ends up there when installed alongside surelog (not separately)?

Reply:
im cool with that hack, its possible that `find_package(UHDM)` may still work/cmake file still installed when its vendored, I can double check the logs

Reply:
Yes, the only thing that ends up in lib is UHDM.capnp. Might be a particular configuration of cmake on that 64 bit system.

Alright, so let's put in this hack and see how things go.